matrix-sdk-android/build.gradle

```diff
@@ -158,7 +158,7 @@ dependencies {
     implementation libs.squareup.moshi
     kapt libs.squareup.moshiKotlin
 
-    api "com.atlassian.commonmark:commonmark:0.13.0"
+    api "com.atlassian.commonmark:commonmark:0.17.0"
```

Comment:
We should change the groupId, see https://github.com/commonmark/commonmark-java/releases/tag/commonmark-parent-0.17.0

Reply:
Does this mean we should wait for` io.notices.markwon` to upgrade its version? Or could we apply a Gradle `resolutionStrategy` for that lib?

Reply:
I'd vote for waiting for markwon to upgrade

As the artifacts use the same class package names we get duplicate class errors when jetifying, we could potentially pre-jetify some of the libraries (and save some build time)


there's also breaking changes in `DollarMathsDelimiterProcessor`, _not tested_

```kotlin
internal class DollarMathsDelimiterProcessor : DelimiterProcessor {
    override fun getOpeningCharacter() = '$'

    override fun getClosingCharacter() = '$'

    override fun getMinLength() = 1

    override fun process(openingRun: DelimiterRun, closingRun: DelimiterRun): Int {
        val isInline = openingRun.length() == 1 && closingRun.length() == 1
        val maths = if (isInline) {
            InlineMaths(InlineMaths.InlineDelimiter.SINGLE_DOLLAR)
        } else {
            DisplayMaths(DisplayMaths.DisplayDelimiter.DOUBLE_DOLLAR)
        }

        var tmp = openingRun.opener.next
        while (tmp != null && tmp !== closingRun.closer) {
            val next = tmp.next
            maths.appendChild(tmp)
            tmp = next
        }
        openingRun.opener.insertAfter(maths)
        return if (isInline) 1 else 2
    }
}
```